src/App.jsx

```diff
@@ -14,45 +17,116 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+export const App = () => {
+  const [listOfGoods, setListOfGoods] = useState([...goodsFromServer]);
+  const [targetInnerText, setTargetInnerText] = useState('');
+  const [isReverse, setIsReverse] = useState(false);
+
+  const sortList = (type) => {
+    let newList;
+
+    switch (type) {
+      case 'Sort alphabetically':
```

Comment:
it's better to move 'Sort alphabetically' and 'Sort by length' to constants as it will make a code cleaner